### ISSUE TITLE
No Dockerfile for core-debian-bookworm

### ISSUE THREAD

**Issue by neonmaus** [kasmtech/workspaces-core-images#41]
Is there a reason why there is no dockerfile available for the core-debian-bookworm image?

**Comment by j-travis** [comment]
It's the main one. https://github.com/kasmtech/workspaces-core-images/blob/develop/dockerfile-kasm-core

We just pass an argument `BASE_IMAGE`  in the build to use the bookwork base.
https://github.com/kasmtech/workspaces-core-images/blob/develop/ci-scripts/template-vars.yaml#L73

Over time this project has become increasingly complex to support more distros so it can be a bit hard to follow
https://github.com/kasmtech/workspaces-core-images/blob/develop/ci-scripts/template-vars.yaml#L73
